pkgs/development/libraries/libuv/default.nix

```diff
@@ -55,6 +59,9 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  # Some of the tests use localhost networking.
+  __darwinAllowLocalNetworking = true;
```

Comment:
Perhaps we should make this true globally? This kind of thing works by default on linux in sandboxing iirc.

Reply:
Unlike on linux localhost networking is not isolated by default so I think this is a good thing. I'll probably have to revisit the sandboxing rules at some point so we could remove the option then, but so far this has not been prevalent enough to be necessary IMHO.